### ISSUE TITLE
Change `checkout` action version

### ISSUE THREAD

**Issue by OmAximani0** [robusta-dev/robusta#682]
Since `action/checkout@v2` uses `Node 12.x.x` and it is [deprecated](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/) I think we must use `action/checkout@v3`

https://github.com/robusta-dev/robusta/blob/f237628424813884f576205d1d9440ffd7bddb95/.github/workflows/release.yaml#L17-L19

**Comment by VamshiReddy02** [comment]
can I take up this issue?

**Comment by OmAximani0** [comment]
Yes @VamshiReddy02 , you can. make sure to change it in every workflow file of the project

**Comment by aantn** [comment]
Hey all, at the moment `action/checkout@v2` still works fine!

Is there a specific deadline by which we need to update?

I would be happy to suggest some other issues too where we're actively looking for contributors.

**Comment by OmAximani0** [comment]
Yes Node 12 is there until [summer of 2023](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/#:~:text=We%20plan%20to%20migrate%20all%20actions%20to%20run%20on%20Node16%20by%20Summer%202023)
